### PR TITLE
Homepage ux update

### PR DESCRIPTION
**Pull request by harshrjjpt**
HOMEPAGE UX UPDATE
1) FULL PAGE
image
2) Ipad AIR
image
3)Ipad MINI
image
4) Iphone XR
image
5)Iphone 12 pro
image
6) Samsung s8+
image

**FOOTER**

7) in footer encapsulated footer left content in left div and right content in right div
8) made flex direction to column in footer when screen is less than 900px
9) aligned right and left content as mentioned in figma .
10) made support line in center and applied space between to graph protocol line and social icons
11) changed all social icons svg

**NAVBAR**

-> first applied changes as given in the figma for full resolution.
-> improved buttons 
-> handled for all resolutions
-> Added shadows to it

**HOMESCREEN**

-> lifted the "Real-time Subgraph Explorer & Visualizer" head above the form box
-> made it to the center of the screen, above the input bar.
-> made info icon to the right side of the input bar
-> made flex direction to column in form box so that input and button get aligned in separate lines
-> applied border radius to both
-> handled the error line
-> changed recent subgraphs UX as mentioned in figma (V-shaped)

### CODE REVIEW

File: src/components/Footer/footer.scss [hunk 1]
```diff
@@ -1,76 +1,157 @@
...
+  position: relative;
   width: 100%;
   height: 50px;
   display: flex;
   justify-content: space-evenly;
   align-items: center;
-  background-color: $primaryColor;
-  color: $colorWhite;
+  background-color: var(--homeBackground);
+  color: $colorBlack;
```
Comment (SumitMish23) [inline src/components/Footer/footer.scss:10]: Add Border shadow in the footer

**Review by tech-nerd — CHANGES_REQUESTED** [review]
image
1. On hover the DOCS color is changed to a dark theme color, which is not the case in the new UX.

image
1. button colors 0f explore button and docs button are different

image
1. After input endpoint, input box background color changes

**Comment by deepakkashyap3013** [comment]
1. Footer upper shadow is not implemented :point_down: 

Screenshot from 2022-08-20 12-12-07

2. When the view height is changed the footer postition changes

Screenshot from 2022-08-20 12-15-38

4. 
- Remove the column Gap 
- Don't use margin 
- Reduce the width of the parent div for Subgraph links ( .subgraph-link-container{...} )

Screenshot from 2022-08-20 12-24-17

5. It should look soemthing like this :point_down: 

Screenshot from 2022-08-20 12-27-01

**Comment by deepakkashyap3013** [comment]
**Don't use margin to manipulate the position of the elelemts**
- Instead Control the postition of the elements through the parent containers.
- If we continue to postion the elements using margin in future maintaining the code is very difficult. 
- Use margins whenever necessary

- Also the Text elements in the capsule are not in perfect center... please fix it

Screenshot from 2022-08-20 12-36-15
Screenshot from 2022-08-20 12-36-25
Screenshot from 2022-08-20 12-36-28
Screenshot from 2022-08-20 12-36-56

**Comment by deepakkashyap3013** [comment]
Screenshot from 2022-08-20 18-21-12

Few Changes :
- Change the width to 888px
- Add the column gap to 50px

Rest Everything looks fine !

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -5,74 +5,77 @@ import { ReactComponent as TwitterSVG } from '../../svg/twitter.svg';
...
-            <GraphProtocolSVG />
-            {Constants.LABELS.commonLables.GRAPH_PROTOCOL}
-          </a>
-        </div>
-        <div className="social-icons-footer">
-          <div className="icon">
-            <a href={Constants.URL.DISCORD} target="_blank" rel="noreferrer">
-              <DiscordSVG />
+      <footer className="main-footer">
+        <div className="left">
```
Comment (tech-nerd) [inline src/components/Footer/Footer.tsx:15]: can we change class name to 'left-container', that will be the more logical name

File: src/components/Footer/Footer.tsx [hunk 3]
```diff
@@ -5,74 +5,77 @@ import { ReactComponent as TwitterSVG } from '../../svg/twitter.svg';
...
+          </div>
         </div>
-
-        <div className="copyright-info">
-          {Constants.LABELS.commonLables.COPYRIGHT}
-          <a href={Constants.URL.DAPPLOOKER_INDEXER} target="_blank" rel="noreferrer">
-            {Constants.LABELS.commonLables.DAPPLOOKER}
-          </a>
-          {Constants.LABELS.commonLables.TEAM}
+        <div className="right">
```
Comment (tech-nerd) [inline src/components/Footer/Footer.tsx:71]: can we change the class name to 'right-container', that will be a more logical name

File: src/components/Home/home.scss [hunk 4]
```diff
@@ -135,105 +184,300 @@
...
-  .search-box {
-    margin: auto;
-    width: 700px;
+@media screen and (max-width: 1000px) {
+  .main {
+    .container {
+      .explore-msg {
+        font-size: 45px;
+      }
+      .error-message {
```
Comment (tech-nerd) [inline src/components/Home/home.scss:193]: why empty classes

File: src/components/Home/home.scss [hunk 5]
```diff
@@ -135,105 +184,300 @@
...
+@media screen and (max-width: 900px) {
+  .main {
+    .container {
+      row-gap: 1px;
+
+      .explore-msg {
+        font-size: 40px;
+        margin-bottom: 30px;
+      }
+      .error-message {
```
Comment (tech-nerd) [inline src/components/Home/home.scss:228]: why empty classes

File: src/components/Home/home.scss [hunk 6]
```diff
@@ -135,105 +184,300 @@
...
+      }
+      .search-box {
+        .search-box-form {
+          width: 790px;
+          .search-input {
+            width: 770px;
+          }
+          .search-button {
```
Comment (tech-nerd) [inline src/components/Home/home.scss:236]: why empty classes

File: src/components/Home/home.scss [hunk 7]
```diff
@@ -135,105 +184,300 @@
...
+          }
+        }
+      }
+    }
+    .subgraph-link-container {
+      width: 790px;
+      .subgraph-link {
+        ul {
+          li {
```
Comment (tech-nerd) [inline src/components/Home/home.scss:245]: why empty classes

File: src/components/Home/home.scss [hunk 8]
```diff
@@ -135,105 +184,300 @@
...
+        font-size: 12px;
+      }
+      .search-box {
+        .search-box-form {
+          width: 490px;
+          .search-input {
+            font-size: 18px;
+            width: 470px;
+          }
+          .search-button {
```
Comment (tech-nerd) [inline src/components/Home/home.scss:353]: why empty classes

File: src/components/Home/home.scss [hunk 9]
```diff
@@ -135,105 +184,300 @@
...
+        column-gap: 50px;
+        .links-tab {
+          .subgraph-link {
+            ul {
+              a {
+                li {
+                  min-width: 80.83px;
+                  height: 19.88px;
+                  font-size: 7px;
+                  .link-text {
```
Comment (tech-nerd) [inline src/components/Home/home.scss:473]: why empty classes

File: src/components/Home/home.tsx [hunk 10]
```diff
@@ -92,6 +98,33 @@ const Home: React.FunctionComponent<RouteComponentProps<any>> = ({ history }) =>
...
               ></input>
+               <span>
+              <CustomTooltip
+                placement="right"
+                title={
+                  <>
+                    <h3 style={{ fontSize: '14px', color: '#fffff' }}>
+                      {commonLables.DOCS_INFO_REF_FIRST}{' '}
+                      <a
+                        className="redirect-to-docs"
```
Comment (tech-nerd) [inline src/components/Home/home.tsx:109]: Code is not formatted

File: src/components/Navbar/navbar.tsx [hunk 11]
```diff
@@ -26,13 +26,11 @@ const Navbar: React.FunctionComponent<{}> = (props) => {
               {theme === label.LIGHT_THEME_LABEL ? (
                 <img
                   src="/images/GraphLooker_theme_color_text.png"
```
Comment (tech-nerd) [inline src/components/Navbar/navbar.tsx:28]: Please format the code

File: src/components/Navbar/navbar.tsx [hunk 12]
```diff
@@ -26,13 +26,11 @@ const Navbar: React.FunctionComponent<{}> = (props) => {
...
-                  height="50px"
                   alt="GraphLooker-icon"
                 ></img>
               ) : (
                 <img
                   src="/images/GraphLooker_white_text.png"
```
Comment (tech-nerd) [inline src/components/Navbar/navbar.tsx:33]: Please format the code

**Review by tech-nerd — CHANGES_REQUESTED** [review]

**Review by tech-nerd — CHANGES_REQUESTED** [review]
image
Footer is not sticking to bottom

**Review by tech-nerd — CHANGES_REQUESTED** [review]

**Review by tech-nerd — APPROVED** [review]

**Pull request merged**